_impls/libassh.md

```diff
@@ -0,0 +1,118 @@
+---
+title: libassh
+homepage: http://libassh.org/
+source-repository: https://hg.savannah.nongnu.org/hgweb/libassh/
+license: "[LGPL 2.1](https://git.libssh.org/projects/libssh.git/tree/COPYING)"
+first-release:
+    date: 2020-09-28
+latest-release:
+    version: 1.0
+    date: 2021-01-03
+#changelog: TODO

```

Comment:
No changelog? That's a pity; it's super helpful for all kinds of things (like... "Wait, you are using version 1.0? Oh but support for protocol XYZ was only added in version 1.2")

Reply:
Yes, I will add a changelog in the next release of the library. There were no protocol change between rc1 and 1.0.